kotest-property/src/commonMain/kotlin/io/kotest/property/arbitrary/array.kt

```diff
@@ -12,3 +12,15 @@ internal inline fun <P, A> Arb.Companion.toPrimitiveArray(
    is Arb -> generateArrayLength
    is Exhaustive -> generateArrayLength.toArb()
 }.flatMap { length -> Arb.list(generateContents, length..length) }.map { it.toArray() }
+
+inline fun <reified A> Arb.Companion.array(
+   gen: Gen<A>,
+   range: IntRange = 0..100,
+   crossinline toArray: Collection<A>.() -> Array<A> = { this.toTypedArray() }
+): Arb<Array<A>> {
+   check(!range.isEmpty())
```

Comment:
Why should it be >= 0? Can't you have ranges from -5 to 5, for example?

Reply:
@LeoColman 

`range` is the length of the array, so isn't it weird that -5 is allowed?

Also, `Arb.list` has same check logic.

https://github.com/kotest/kotest/blob/c32ca997517a6f7f02302913eb6a5cd00f044c69/kotest-property/src/commonMain/kotlin/io/kotest/property/arbitrary/collections.kt#L93

Reply:
@LeoColman What do you think of my answer?

Reply:
Sorry for not replying. Yeah, it's fine